corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -579,12 +581,15 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 templateName: "case-list-template",
                 useTiles: false,
                 hasNoItems: this.hasNoItems,
-                noItemsText: this.options.collection.noItemsText,
+                noItemsText: this.noItemsText,
                 sortIndices: this.options.sortIndices,
                 selectedCaseIds: this.selectedCaseIds,
                 isMultiSelect: false,
                 showMap: this.showMap,
                 columnStyle: this.columnStyle(),
+                sidebarEnabled: this.options.sidebarEnabled,
+                triggerEmptyCaseList: this.options.triggerEmptyCaseList,
+                splitScreenVisible: this.options.sidebarEnabled || this.options.triggerEmptyCaseList,
```

Comment:
Is it possible for `triggerEmptyCaseList` to be true if `sidebarEnabled` is false? [This code](https://github.com/dimagi/commcare-hq/pull/33060/files#diff-cd3ce94b57385fe87dc296bec57917470f84399b356c8b975c74b7f4d3fe1c01R112-R114) makes me think it isn't (I realize that `sidebarEnabled` in that code is a local var, not an attribute coming from formplayer, but I'm assuming that the word "sidebarEnabled" means the same thing in both places).